Taller152/src/paqueteuno/EscribirArchivo.java

```diff
@@ -5,12 +5,36 @@
  */
 package paqueteuno;
 
-import paquetetres.*;
+import java.util.Formatter;
 
 /**
  *
  * @author reroes
  */
 public class EscribirArchivo {
-    
-}
+    public static void hola() {
+        Formatter archivo = null;
+        try {
+            String ruta = "/data/archivo1.txt";
+            String[] paises = {"Ecuador", "Colombia", "Perú", "Venezuela", 
+                "Chile", "Bolivia"};
+            String[] capitales = {"Quito", "Bogotá", "Lima", "Caracas", 
+                "Santiago", "La Paz"};
+            int[] poblacion = {17643054, 51152371, 33050325, 28435940, 19116201,
+                11673021};
+            String cadena = "";
+            archivo = new Formatter("data/archivo1.txt");
+            for (int i = 0; i < paises.length; i++){
+                archivo.format("%s %s, con capital %s tiene una poblacion"
+                        + " de %d\n", cadena, paises[i], 
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.